pkg/cmd/preview/preview.go

```diff
@@ -62,7 +67,7 @@ func (o *PreviewOptions) AddPreviewFlags(cmd *cobra.Command) {
 		i18n.T("Automatically show plan details with interactive options"))
 	cmd.Flags().BoolVarP(&o.All, "all", "a", false,
 		i18n.T("Automatically show all plan details, combined use with flag `--detail`"))
-	cmd.Flags().BoolVarP(&o.NoStyle, "no-style", "", false,
+	cmd.Flags().BoolVarP(&o.NoStyle, "no-style", "s", false,
```

Comment:
Using "s" as an abbreviation for "no-style" is confusing.

Reply:
That's true. I have deleted the short hand letter for "no-style" flag and updated the examples in relevant docs and codes.